test/swar/BasicOperations.cpp

```diff
@@ -41,6 +41,15 @@ static_assert(
     multiplication_OverflowUnsafe_SpecificBitCount<3>(Micand, Mplier).value()
 );
 
+TEST_CASE("Jamie's totally working exponentiation :D") {
+    constexpr auto base     = SWAR<8, u32>{0x02'01'05'06}; //   2 | 1 |  5 |   6
+    constexpr auto exponent = SWAR<8, u32>{0x07'00'02'03}; //   7 | 0 |  2 |   3
+    constexpr auto expected = SWAR<8, u32>{0x80'01'19'D8}; // 128 | 1 | 25 | 216
```

Comment:
Am I reviewing code made by ChatGPT?
It's technically correct to substitute the undefined 0^0 by the perfectly defined 1^0.  Still, aren't there more useful examples? Like 3^5 = 243?
Also, think about this suggestion:
`expected = SWAR<8, u32>{(127 << 24) + (1 << 16) + (25 << 8) + (216 << 0)}`
help us with some notation, macros may be considered, to initialize the lanes of a SWAR.
The explicit `meta::BitmaskMaker<....>::value` has been very tedious, error prone, time consuming for just repeating a constant, let alone initialize lanes to dissimilar values.